docs/mlir_interoperation.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>I would personally make this example smaller by removing the f line. I feel that it does not really add to the understanding and only makes the example bigger.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
totally agree